## Device settings, provide one of the following:
##  - device_id (most secure if you use multiple devices)
##  - port + baudrate information
##  - nothing: the driver scans for devices and selects the first found.
# device_id: '077007EF' # uppercase hex string
# port: '/dev/ttyUSB0'
# baudrate: 921600 # non necessary for some devices

## Log file (optional), placed in ~/.ros/ otherwise use absolute path
# log_file: log.mtb

publisher_queue_size: 5

# TF transform frame_id (default: imu_link), you may want to change it if you use multiple devices
#frame_id: "imu_link"

# Message publishers
pub_imu: true
pub_quaternion: false
pub_mag: false
pub_angular_velocity: false
pub_acceleration: false
pub_free_acceleration: false
pub_dq: false
pub_dv: false
pub_sampletime: true
pub_temperature: false
pub_pressure: false
pub_gnss: false
pub_twist: false
pub_transform: false
pub_positionLLA: false
pub_velocity: false


## Sensor standard deviation [x,y,z] (optional)
## This value is used to override the covariance matrix in sensor_msgs/Imu and
## sensor_msgs/MagneticField messages.
# linear_acceleration_stddev: [0, 0, 0] # [m/s^2]
# angular_velocity_stddev: [0, 0, 0] # [rad/s]
# orientation_stddev: [0, 0, 0] # [rad]
# magnetic_field_stddev: [0, 0, 0] # [Tesla]
